newIDE/app/src/SceneEditor/EventsBasedObjectScenePropertiesDialog.js

```diff
@@ -0,0 +1,214 @@
+        <LineStackLayout expand noMargin>
+          <TextField
+            floatingLabelText={<Trans>Left bound</Trans>}
+            fullWidth
+            type="number"
+            value={areaMinX}
+            onChange={(e, value) => setAreaMinX(parseFloat(value) || 0)}
+            errorText={
+              areaMinX > areaMaxX ? (
+                <Trans>Left bound should be smaller than right bound</Trans>
+              ) : null
+            }
+          />
+          <TextField
+            floatingLabelText={<Trans>Right bound</Trans>}
+            fullWidth
+            type="number"
+            value={areaMaxX}
+            onChange={(e, value) => setAreaMaxX(parseFloat(value) || 0)}
+            errorText={
+              areaMinX > areaMaxX ? (
+                <Trans>Right bound should be greater than left bound</Trans>
+              ) : null
+            }
+          />
+        </LineStackLayout>
+        <LineStackLayout expand noMargin>
+          <TextField
+            floatingLabelText={<Trans>Top bound</Trans>}
+            fullWidth
+            type="number"
+            value={areaMinY}
+            onChange={(e, value) => setAreaMinY(parseFloat(value) || 0)}
+            errorText={
+              areaMinY > areaMaxY ? (
+                <Trans>Top bound should be smaller than bottom bound</Trans>
+              ) : null
+            }
+          />
+          <TextField
+            floatingLabelText={<Trans>Bottom bound</Trans>}
+            fullWidth
+            type="number"
+            value={areaMaxY}
+            onChange={(e, value) => setAreaMaxY(parseFloat(value) || 0)}
+            errorText={
+              areaMinY > areaMaxY ? (
+                <Trans>Bottom bound should be greater than right bound</Trans>
+              ) : null
+            }
+          />
+        </LineStackLayout>
+        <LineStackLayout expand noMargin>
+          <TextField
+            floatingLabelText={<Trans>Z min bound</Trans>}
+            fullWidth
+            type="number"
+            value={areaMinZ}
+            onChange={(e, value) => setAreaMinZ(parseFloat(value) || 0)}
+            disabled={!isRenderedIn3D}
+            errorText={
+              areaMinZ > areaMaxZ ? (
+                <Trans>Z min bound should be smaller than Z max bound</Trans>
+              ) : null
+            }
+          />
+          <TextField
+            floatingLabelText={<Trans>Z max bound</Trans>}
+            fullWidth
+            type="number"
+            value={areaMaxZ}
+            onChange={(e, value) => setAreaMaxZ(parseFloat(value) || 0)}
+            disabled={!isRenderedIn3D}
+            errorText={
+              areaMinZ > areaMaxZ ? (
+                <Trans>Z max bound should be greater than Z min bound</Trans>
+              ) : null
+            }
+          />
+        </LineStackLayout>
+        <LineStackLayout expand noMargin>
+          <Checkbox
+            label={<Trans>Use 3D rendering</Trans>}
```

Comment:
Let's discuss this, but this seems a setting too important to be just called "Use 3D rendering". I think it could/should be a choice you do when you create a new object, where you're giving a dialog showing "3D object" or "2D object", and this can't be modified later (unless you remove all children objects).